st2mistral/functions/stackstorm.py

```diff
@@ -18,16 +18,104 @@
 from oslo_config import cfg
 from oslo_log import log as logging
 
-from st2mistral import config
-config.register_opts()
-
 from mistral import exceptions as exc
+from st2common.jinja.filters import crypto
```

Comment:
I thought we agreed to duplicate the Jinja filters in st2mistral. I'm against including st2common here as a dependency. This will complicate the deployment story for Mistral.